webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala

```diff
@@ -222,7 +222,7 @@ class NodeGroupForm(
       & "group-save" #> { if (CurrentUser.checkRights(AuthorizationType.Group.Edit))
                     <span class="save-tooltip-container">
                       {
-                        SHtml.ajaxButton(<span>Save <i class="fa fa-download"></i></span>, onSubmit _)  %  ("id" -> saveButtonId) % ("class" -> " btn btn-success btn-icon")
+                        SHtml.ajaxSubmit("Save", onSubmit _)  %  ("id" -> saveButtonId) % ("class" -> " btn btn-success")
```

Comment:
Not an easy one for me, I'm still not so sure about how it works, but it seems to resolve the bug
This is how it was done in 7.0

Reply:
You need to keep the element as before, just replace ajaxButton by ajaxSubmit

Reply:
Submit, applies all changes in the form before executing it, Ajaxbutton executes it without the changes from the form. so it explains why the changes were not kept

Reply:
W

> You need to keep the element as before, just replace ajaxButton by ajaxSubmit

We can't give HTML to `ajaxSubmit` like `ajaxButton` (for the icon), we need to find another way to add the dowload icon at the end of the button